deployment/ansible/roles/install_config_nginx/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install Nginx
+  ansible.builtin.dnf:
+    name: nginx
+    state: present
+    update_cache: true
+
+- name: Create Nginx configuration file
+  ansible.builtin.template:
+    src: nginx.conf.j2
+    dest: /etc/nginx/conf.d/default.conf
```

Comment:
not an issue, but i would  prefer use old style configs with sites-enabled/sites-available.